pyccel/codegen/printing/cucode.py

```diff
@@ -61,14 +61,45 @@ def _print_Module(self, expr):
             else:
                 local_imports += self._print(imp)
 
-        imports = f'{c_headers_imports}\
-                    extern "C"{{\n\
-                    {local_imports}\
-                    }}'
+        imports = f'{c_headers_imports}\n\
+                    {local_imports}\n'
 
         code = f'{imports}\n\
                  {global_variables}\n\
                  {body}\n'
 
         self.exit_scope()
         return code
+
+    def _print_ModuleHeader(self, expr):
+        self.set_scope(expr.module.scope)
+        self._in_header = True
+        name = expr.module.name
+        if isinstance(name, AsName):
+            name = name.name
+        # TODO: Add interfaces
+
+        funcs = ""
+        cuda_headers = ''
```

Comment:
Can we pick `""` or `''` instead of having both over 2 lines

Reply:
fixed 3580b26